### ISSUE TITLE
Uncaught TypeError: this._element.openDevTools is not a function

### ISSUE THREAD

**Issue by PhoenixDimazzz** [atom-community/markdown-preview-plus#490]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.47.0 x64
**Electron**: 5.0.13
**OS**: Unknown Windows version
**Thrown From**: [markdown-preview-plus](https://github.com/atom-community/markdown-preview-plus) package 3.11.5

### Stack Trace

Uncaught TypeError: this._element.openDevTools is not a function

```
At C:\Users\FORMOZA 44-88-44\.atom\packages\markdown-preview-plus\lib\markdown-preview-view\webview-handler.js:158

TypeError: this._element.openDevTools is not a function
    at WebviewHandler.openDevTools (/packages/markdown-preview-plus/lib/markdown-preview-view/webview-handler.js:158:23)
    at /packages/markdown-preview-plus/lib/markdown-preview-view/markdown-preview-view.js:148:30)
    at CommandRegistry.handleCommandEvent (~/AppData/Local/atom/app-1.47.0/resources/app/static/<embedded>:11:349576)
    at CommandRegistry.dispatch (~/AppData/Local/atom/app-1.47.0/resources/app/static/<embedded>:11:348051)
    at AtomEnvironment.dispatchContextMenuCommand (~/AppData/Local/atom/app-1.47.0/resources/app/static/<embedded>:1:765638)
    at EventEmitter.t (~/AppData/Local/atom/app-1.47.0/resources/app/static/<embedded>:1:773644)
    at EventEmitter.emit (events.js:194:13)
```

### Commands

```
  2x -2:51.9.0 markdown-preview-plus:toggle (input.hidden-input)
     -1:21.5.0 markdown-preview-plus:sync-source (webview.markdown-preview-plus.native-key-bindings)
     -1:06.8.0 markdown-preview-plus:toggle (input.hidden-input)
     -0:39.6.0 markdown-preview-plus:select-syntax-theme (input.hidden-input)
     -0:32.5.0 pane:reopen-closed-item (input.hidden-input)
     -0:13.7.0 markdown-preview-plus:open-dev-tools (webview.markdown-preview-plus.native-key-bindings)
```

### Non-Core Packages

```
advanced-open-file 0.16.8 
atom-beautify 0.33.4 
atom-clock 0.1.18 
atom-i18n 0.23.1 
auto-encoding 0.7.2 
file-icons 2.1.41 
indent-guide-improved 1.4.13 
markdown-preview-plus 3.11.5 
markdown-writer 2.11.10 
medium-export 0.4.0 
pandoc-convert 1.1.1 
pandoc-pdf 0.1.0 
tool-bar 1.3.0 
```

**Comment by lierdakil** [comment]
See https://github.com/atom-community/markdown-preview-plus/issues/487

TL;DR: Upstream broke Atom 1.47. Downgrade to 1.46 for now. 1.48 should be fine once it comes out eventually.
